### ISSUE TITLE
Baseline: journey from search to works and back again

### ISSUE THREAD

**Issue by taceybadgerbrook** [wellcomecollection/wellcomecollection.org#8766]
**What is the problem to solve?**
We want to know whether the concepts pages are helping researchers find what they are looking for.

**What stage of research is it? (discovery/general usability testing/post release)**
discovery - baseline

**What are the objectives?**
What is average user behaviour? 
- how many journeys search -> works page -> search

**What are the assumptions?**
We don't need to know how many times they search or how many works pages they view.

**What research technique is required?**
Python

**Comment by taceybadgerbrook** [comment]
@jennpb Just to check, is this to see if creating the concepts pages cut down on the search/works page/search journeys? or is it about the positioning of item view beside works page information?

**Comment by jennpb** [comment]
I don't think the concepts pages play into this at all.

I would rephrase the problem to solve as 'Understand paths users take when navigating between search and works pages and back to search'

This would likely pick up these journeys:

Search > Search results > Work > (maybe Item) > Search results > Work etc
Search > Search results > Work > Search > Search results > Work etc
and maybe other anomalies like going back to Collections

**Comment by taceybadgerbrook** [comment]
I think it would be better if this analysis was folded into #8827.

**Comment by jennpb** [comment]
Yes, that looks right to me

**Comment by taceybadgerbrook** [comment]
These journeys have been incorporated into #8827
